src/devices/src/virtio/balloon/device.rs

```diff
@@ -464,7 +464,8 @@ pub(crate) mod tests {
 
     use super::super::CONFIG_SPACE_SIZE;
     use super::*;
-    use crate::virtio::queue::tests::*;
+    use crate::virtio::test_utils::VirtQueue;
```

Comment:
Ballon device has some testing utilites that we may want to move in a separate module, right?

Reply:
Pay attention that `default_mem` is already present in `virtio/test_utils`.